src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -472,6 +538,49 @@ define(function (require) {
     };
 
     /**
+     * get bounds of features from geoJson
+     *
+     * @method getBounds
+     * @param mapData {Object}
+     * @return {undefined}
+     */
+    TileMap.prototype.getBounds = function (mapData) {
+      var bounds = L.geoJson(mapData).getBounds();
+      return bounds;
+    };
+
+    /**
+     * if attribute is checked/true
+     • normalize data for heat map intensity
+     *
+     * @param mapData {Object}
+     * @param nax {Number}
+     * @method dataToHeatArray
+     * @return {undefined}
```

Comment:
Return undefined? This appears to return a 3 item array. A descriptive variable might be helpful at very least for the 3rd item in that array. Could probably shorten this function by making that descriptive and not duplicating the entire array for each return?